control-plane/agents/ha-node/src/main.rs

```diff
@@ -47,7 +48,15 @@ async fn main() {
         .expect("Expect to be initialized only once");
 
     cluster_agent_client()
-        .register(cli.node_name.clone(), cli.grpc_endpoint)
+        .register(&NodeAgentInfo::new(
+            cli.node_name.clone(),
+            cli.grpc_endpoint
```

Comment:
grpc_endpoint should be a `SocketAddress`

Reply:
yes. I've kept it as Url to keep it consistent with other binaries.